### DISCUSSION TITLE
Intel AX210NGW wifi/bt card

### DISCUSSION THREAD

**Discussion by controlfix87** [osy86/HaC-Mini#927] [Q&A] [answered]
I’ve upgraded my Wifi card to Intel AX210NWG from Intel 8265NGW and it worked great.
I’m running on BigSur 11.5.2 (latest) and it seems that the card is not getting a MAC address nor I can’t select any network (not showing any).
Installed the latest HacMini pkg 3.1.1 and selected intel wifi/bt, reset nvram twice.
no luck on that.
can you please help out to get my wifi working again?

**Comment by goodbest** [comment, marked as answer]
You should head over to the original project repo. https://github.com/OpenIntelWireless

The AX210 is not supported yet in v2.0 stable (used in Hac mini 3.1.1). https://openintelwireless.github.io/itlwm/Compat.html#mvm-gen-2-iwx

It seems that AX210 is partially supported in v2.1 alpha build.
 https://github.com/OpenIntelWireless/itlwm/releases

**Reply by controlfix87** [reply]
Thanks! will follow this one :)

**Reply by controlfix87** [reply]
Is the AX200 supported?...
